core/src/main/java/com/linecorp/armeria/common/logging/ClientConnectionTimingsBuilder.java

```diff
@@ -107,8 +107,9 @@ public ClientConnectionTimingsBuilder pendingAcquisitionStart() {
      * @throws IllegalStateException if {@link #pendingAcquisitionStart()} is not invoked before calling this.
      */
     public ClientConnectionTimingsBuilder pendingAcquisitionEnd() {
-        checkState(pendingAcquisitionStartTimeMicros >= 0, "pendingAcquisitionStart() is not called yet.");
-        checkState(!pendingAcquisitionEndSet, "pendingAcquisitionEnd() is already called.");
+        // TODO: decide how to handle pendingAcquisitionEnd now that it might be called multiple times
+        // checkState(pendingAcquisitionStartTimeMicros >= 0, "pendingAcquisitionStart() is not called yet.");
+        // checkState(!pendingAcquisitionEndSet, "pendingAcquisitionEnd() is already called.");
```

Comment:
I guess we can prevent calling `pendingAcquisitionEnd()` if `acquireLater()` will be called?